### PR TITLE
Added a new reload function to recache data

### PR DESCRIPTION
**Pull request by Vinochs**
[Favro Card](https://favro.com/organization/c9668dc39f1bd49fed254201/dd23b494d608926aa6d7878d?card=Cre-13365)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@creately/rxdata",
-  "version": "5.1.3",
+  "version": "5.1.4",
```
Comment (Damith88) [inline package.json:3]: package-lock.json should be updated as well.

**Comment by Damith88** [comment]
please update the change log as we are going to make a new release.

File: package.json (hunk 1 above)
Comment (Vinochs) [reply package.json:3]: Updated

File: src/__tests__/collection.spec.ts [hunk 2]
```diff
@@ -449,4 +449,17 @@ describe('Collection', () => {
 
     it('should emit the removed document as a change (remote)');
   });
+
+  describe('reload', () => {
+    it('should load documents and emit alldocs', async done => {
```
Comment (Damith88) [inline src/__tests__/collection.spec.ts:454]: `sync done` is deprecated, and should be avoided

Comment (Vinochs) [reply src/__tests__/collection.spec.ts:454]: But this has been used everywhere in the test. You want me to refactor the tests?

**Review by Damith88 — APPROVED** [review]

**Pull request merged**